### DISCUSSION TITLE
Logins for different users on YT music.

### DISCUSSION THREAD

**Discussion by MysticNynja** [music-assistant/support#1046] [Feature requests and ideas]
I have a family plan for YT Music and my suggestions are getting crazy with my families music.  is there a way to associate logins in HA with different users on YT?

**Comment by OzGav** [comment]
In the next version there will be some multi account support but I don’t know if it is extending to YT. Keep your eyes out for the update and see if it meets your requirements.

**Reply by MysticNynja** [reply]
if anything I can take a look once the multi account support hits and see if I can add it :) Thanks!

**Comment by OzGav** [comment]
V1 question. V2 supports multiple accounts.
